examples/blob/index.ts

```diff
@@ -3,14 +3,14 @@
 import * as azure from "@pulumi/azure";
 
 const resourceGroup = new azure.core.ResourceGroup("resourcegroup", {
-    location: "West US 2",
+    location: azure.Locations.WestUS2,
 });
 
 // Create a storage account for our images
 const storageAccount = new azure.storage.Account("storage", {
     resourceGroupName: resourceGroup.name,
-    accountReplicationType: "LRS",
-    accountTier: "Standard",
+    accountReplicationType: azure.storage.AccountReplicationTypes.LRS,
+    accountTier: azure.storage.AccountTiers.Standard,
```

Comment:
That's a lot more characters than before. If we can enforce the values in types (see my other comment), wouldn't it be okay to keep "LRS" and "Standard"? people would still get a compile error if they mistype it.
Might be a subjective choice - but we'd need to pick a better one for all the examples.